students/AndrewKim/lesson04/mailroom_part2.py

```diff
@@ -0,0 +1,49 @@
+
+donor_history = {'Andrew Kim': [5.0, 3.0], 'Jamie Park': [4.0], 'Tim Duncan': [3.0], 'Billy Yan': [2.0]}
+
+def thank_you():
+    #prompt for a Full Name
+    name = input("Type the donor's full name: ")
+    #if user types 'list'
+    while name == "list":
+        for a in donor_history,keys():
```

Comment:
You've got a comma here instead of a period in the call to `keys()` which causes the program to crash. But this can just be `for a in donor_history:` because the `in` operator checks for key membership in dict